README.md

```diff
@@ -56,13 +56,23 @@ See the [Python Plugin API](http://pynvim.readthedocs.io/en/latest/usage/python-
 
 #### Development
 
-If you change the code, you need to run
-```sh
-pip2 install .
-pip3 install .
-```
-for the changes to have effect. For instructions of testing and troubleshooting,
-see the [development](http://pynvim.readthedocs.io/en/latest/development.html) documentation.
+Use (and activate) a local virtualenv.
+
+    python -m virtualenv --python=python3 venv
```

Comment:
Is `venv` the name of a specifiic virtualenv? A bit unfortunate as `venv` is a basic virtualenv implementation shipped with python.

Reply:
That's the path.
I would rather use `.venv` instead, and the `venv` module (with py3): `python -m venv .venv`

Reply:
@blueyed Thanks. Is there a way to specify different versions with `python -m venv`? 

```
$ python3 -m venv --python=python3.7 py37                                                                                                                   
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]      
            [--upgrade] [--without-pip] [--prompt PROMPT]        
            ENV_DIR [ENV_DIR ...]                                              
venv: error: unrecognized arguments: --python=python3.7                          
```

Reply:
It uses (only) the existing one (the caller) I assume.